emmet-builders/emmet/builders/mobility_screening/migration_graph.py

```diff
@@ -0,0 +1,64 @@
+class MigrationGraphBuilder(MapBuilder):
+    def __init__(
+        self,
+        materials: MongoStore,
+        electrodes: MongoStore,
+        tasks: MongoStore,
+        target: MongoStore,
+        working_ion_entry: ComputedEntry,
+        ltol: float = 0.4,
+        stol: float = 0.6,
+        angle_tol: float = 15,
+        **kwargs
+    ):
+        self.materials = materials
+        self.electrodes = electrodes
+        self.tasks = tasks
+        self.working_ion_entry = working_ion_entry
+        self.ltol = ltol
+        self.stol = stol
+        self.angle_tol = angle_tol
+        self.target = target
+        
+        super().__init__(source=electrodes, target=target, **kwargs)
+        self.sources.append(tasks)
+    
+    def get_items(self):
+        for item in super(MigrationGraphBuilder,self).get_items():
+            tds = list(self.tasks.query({"task_id":{"$in":item["material_ids"]}}))
+            item.update({"task_docs":tds})
+            yield item
+
+	#get structure entry from task store and attach migration graph dict to item    
+    def unary_function(self, item):
+        new_item = dict(item)
+        task_documents = [TaskDocument.parse_obj(td) for td in new_item["task_docs"]]
+        entries = [task_doc.structure_entry for task_doc in task_documents]
+        mg = get_migration_graph(compatibility.process_entries(entries), 
+                                 self.working_ion_entry, 
+                                 ltol=self.ltol, stol=self.stol, angle_tol=self.angle_tol)
+        new_item["mg"] = mg
+        return new_item
+    
+    def get_migration_graph(cses, wi_entry, max_distance_min=0.1, max_distance_max=10, ltol=0.4, stol=0.6, angle_tol=15) -> dict:
```

Comment:
Can you add docstrings and specify input types for your functions? We want to be consistent with the rest of emmet. Here is one example for reference: https://github.com/materialsproject/emmet/blob/main/emmet-builders/emmet/builders/materials/electrodes.py#L51

If you are unfamiliar with typing here is an explanation and we can meet next week to go over it if needed! https://docs.python.org/3/library/typing.html